ddtrace/contrib/logging/__init__.py

```diff
@@ -40,15 +40,15 @@ def hello():
 If you prefer to instrument manually, patch the logging library then update the
 log formatter as in the following example
 
-Note that ``dd.trace_id`` and ``dd.span_id`` must come last::
+Note that the ``dd.trace_id`` and ``dd.span_id`` entries must come last::
```

Comment:
should we remove this and write it as a "this is exactly how to do it", e.g. don't call out that these have to be last, just say... "you *must* add this string to your format string"